docs/editing-components.md

```diff
@@ -335,7 +336,26 @@ that includes the `RelatedContent` molecule in one of its StreamFields.
 ## How-to guides
 
 
-### Adding a field
+### Creating a new component
+
+1. Review the [Notes on Atomic Design](../atomic-structure/) page.
+1. Add each of the parts mentioned above:
+   1. [Create the Python class](#the-python-class)
```

Comment:
If this is intended to be a sub-list, [it needs to be indented by four spaces](https://commonmark.org/help/tutorial/10-nestedLists.html). Currently this shows up as one list for me.

Reply:
Gah. That MkDocs markdown renderer strikes again!